packages/react-router/lib/dom/ssr/components.tsx

```diff
@@ -632,28 +651,36 @@ export type ScriptsProps = Omit<
 };
 
 /**
-  Renders the client runtime of your app. It should be rendered inside the `<body>` of the document.
-
-  ```tsx
-  import { Scripts } from "react-router";
-
-  export default function Root() {
-    return (
-      <html>
-        <head />
-        <body>
-          <Scripts />
-        </body>
-      </html>
-    );
-  }
-  ```
-
-  If server rendering, you can omit `<Scripts/>` and the app will work as a traditional web app without JavaScript, relying solely on HTML and browser behaviors.
-
-  @category Components
+ * Renders the client runtime of your app. It should be rendered inside the
+ * `<body>` of the document.
+ *
+ * If server rendering, you can omit `<Scripts/>` and the app will work as a
+ * traditional web app without JavaScript, relying solely on HTML and browser
+ * behaviors.
+ *
+ * @example
+ * import { Scripts } from "react-router";
+ *
+ * export default function Root() {
+ *   return (
+ *     <html>
+ *       <head />
+ *       <body>
+ *         <Scripts />
+ *       </body>
+ *     </html>
+ *   );
+ * }
+ *
+ * @public
+ * @category Components
+ * @mode framework
+ * @param props Props for the
```

Comment:
@brophdawg11 This will be shown as a prop called `props` in the docs, but I guess we want to have a general description instead?

Reply:
yeah - maybe we go with `scriptProps` and clarify in the description

Reply:
good catches - tried to address these in https://github.com/remix-run/react-router/pull/14038